executor/write.go

```diff
@@ -980,6 +1028,9 @@ func (e *InsertValues) initDefaultValues(row []types.Datum, hasValue []bool, ign
 			// Just leave generated column as null. It will be calculated later
```

Comment:
Should we add a comment for initDefaultValues
to distinguish it with fillDefaultValues easily.